target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -186,6 +186,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:wan" "2:lan:3" "3:lan:2"
 		;;
+	engenius,enh202-v1)
+		ucidef_set_interfaces_lan_wan "eth1" "eth0"
```

Comment:
Just
ucidef_set_interface_wan "eth0"
here, the lan is in the switch statement.

Reply:
Yeah I believe the switch is required and eth1 is not possible as a LAN port...

however what if I added `simple-mfd` to eth1 compatible?

Reply:
You are using eth1 in ucidef_add_switch, so specifying it in ucidef_set_interfaces_lan_wan is _redundant_.

If you "disable" eth1 with simple-mfd, then only your wan-port will be working at all (unless your setup is wrong entirely)

Reply:
I was just wondering if there was a way to implement your original suggestion above

> No, the switchdev is better, and it matches the port you specify in 02_network. However, I'd use
> ucidef_set_interfaces_lan_wan "eth1" "eth0"
> in 02_network.

But it seems like there is no way to make the eth1 switch with only 1 port into eth1 instead of eth1.1